### PR TITLE
Make input fileds highlight their contents on focus

### PR DESCRIPTION
**Pull request by humphd**
https://webmademovies.lighthouseapp.com/projects/65733-popcorn-maker/tickets/1142-clicking-editor-textboxes-should-focus-and-highlight-content

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @jbuck received. Current queue size: 0

Live output at: http://butyr.org:8000/d64b8534833633b/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://butyr.org:8000/d64b8534833633b/output.txt

Total script time: 0.57 mins

Server is running on http://pr281.butyr.org/templates/test.html

File: editors/default-editor.html [hunk 1]
```diff
@@ -142,6 +154,9 @@
                 field.addEventListener( "change", function( e ){
                   sendData( false );
                 }, false );
+
+                // When field is clicked, highlight everything within
+                field.addEventListener( "focus", focus, false);
```
Comment (jbuck) [inline editors/default-editor.html]: Change the focus method to highlight

**Comment by jbuck** [comment]
And this doesn't work correctly in Chrome for some reason. Hooray browser differences!

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_cleanup** from @jbuck received. Current queue size: 0

Live output at: http://butyr.org:8000/9a34e70bd1d93b6/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://butyr.org:8000/9a34e70bd1d93b6/output.txt

Total script time: 1.33 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_cleanup** from @jbuck received. Current queue size: 0

Live output at: http://butyr.org:8000/0351119a6a80fcf/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://butyr.org:8000/0351119a6a80fcf/output.txt

Total script time: 0.52 mins

**Pull request closed without merging**